javascript-arrays/main.js

```diff
@@ -0,0 +1,20 @@
+const colors = ['red',
+  'white',
+  'blue'];
+console.log('value of colors [0]: ', colors[0]);
+console.log('value of colors [1]: ', colors[1]);
+console.log('value of colors [2]: ', colors[2]);
+console.log('America is', colors[0], ',', colors[1], 'and', colors[2]);
+colors[2] = 'green';
+console.log('Mexico is', colors[0], ',', colors[1], 'and', colors[2]);
+console.log('the value of colors is: ', colors);
+const students = ['number1',
+  'number2',
+  'number3',
+  'number4'];
+const numberOfStudents = students.length;
+console.log('there are ', numberOfStudents, ' students in the class');
+const lastIndex = numberOfStudents - 1;
+const lastStudent = students[lastIndex];
+console.log('the last student is ', lastStudent);
```

Comment:
Compare this output to the output in the instructions. It doesn't match the sentence exactly.